File: Models/Room.cs
using System.Collections;
using System.Collections.Generic;
using System.ComponentModel.DataAnnotations;
using HotChocolate;
using Microsoft.EntityFrameworkCore;

namespace hospitalGQL.Models
{
    public class Room
    {
        [Key] 
        public long room_number { get; set; }
        public string room_type { get; set; }
        public string status { get; set; }
   
            

        public ICollection<Patient> Patients { get; set; } = new List<Patient>();
        
    }
}

File: src/main/java/com/example/resthospital/Room/Room.java
package com.example.resthospital.Room;

import com.fasterxml.jackson.annotation.JsonProperty;

public class Room
{

  @JsonProperty("room_number")
  int room_number;

  @JsonProperty("room_type")
  String room_type;

  @JsonProperty("status")
  String status;




}
